### PR TITLE
OPENEUROPA-1630: Update drupal-module-template with latest practices

### PR DESCRIPTION
**Pull request by dxvargas**
### Description

Add yml to list of phpcs triggers.

### Change log

- Added: yml extensions to list of phpcs triggers.
- Changed: Composer dependencies.

### CODE REVIEW

File: drupal_module_template.info.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Drupal module template
+name: OpeneEuropa template
```
Comment (sergepavle) [inline drupal_module_template.info.yml]: Typo

File: grumphp.yml.dist [hunk 2]
```diff
@@ -2,21 +2,16 @@ imports:
   - { resource: vendor/openeuropa/code-review/dist/drupal-conventions.yml }
 parameters:
   tasks.phpcs.ignore_patterns:
-    - vendor/
-    - node_modules/
     - build/
+    - node_modules/
+    - vendor/
```
Comment (sergepavle) [inline grumphp.yml.dist]: If I am not wrong, It is make sense to add also this in ignore list: `bower_components`

Comment (dxvargas) [reply grumphp.yml.dist]: I don't think many modules will use that.
It's more a theme related tool.

File: drupal_module_template.info.yml (hunk 1 above)
Comment (dxvargas) [reply drupal_module_template.info.yml]: Fixed.

File: grumphp.yml.dist [hunk 3]
```diff
@@ -2,21 +2,16 @@ imports:
...
-  extra_tasks:
-    phpparser:
-      ignore_patterns: %tasks.phpcs.ignore_patterns%
-      visitors:
-        declare_strict_types: ~
-      triggered_by: %tasks.phpcs.triggered_by%
   extensions:
-    - OpenEuropa\CodeReview\ExtraTasksExtension
\ No newline at end of file
+    - OpenEuropa\CodeReview\ExtraTasksExtension
```
Comment (drupol) [inline grumphp.yml.dist]: This line can go away as well.

File: grumphp.yml.dist [hunk 4]
```diff
@@ -2,21 +2,16 @@ imports:
...
   tasks.phpcs.triggered_by:
-    - php
-    - module
     - inc
-    - theme
     - install
+    - module
+    - php
+    - theme
+    - yml
```
Comment (drupol) [inline grumphp.yml.dist]: Wasn't this line creating issue with yml files ?

Comment (drupol) [reply grumphp.yml.dist]: Fixed while discussing together.
This was creating issues with phpparser, which has been removed and superseded by the new tools in the code-review component.

File: grumphp.yml.dist (hunk 3 above)
Comment (dxvargas) [reply grumphp.yml.dist]: Done.

**Pull request merged**